<?xml version="1.0" ?>

<launch>
    <arg name="rviz" default="true"/>

    <include file="$(find cbf_controllers)/launch/common.launch"/>
    <include file="$(find cbf_controllers)/test/obstacle_simple.launch"/>

    <node name="controller_loader" pkg="controller_manager" type="controller_manager" output="screen"
          args="spawn
          controllers/dcbf_duality_controller
"/>

    <node pkg="legged_controllers" type="legged_target_trajectories_publisher" name="legged_robot_target"
          output="screen"/>

    <group if="$(arg rviz)">
        <arg name="rvizconfig" default="$(find cbf_controllers)/config/rviz.rviz"/>
        <node pkg="rviz" type="rviz" name="rviz" args="-d $(arg rvizconfig)" output="screen"/>
    </group>
</launch>
